setup.py

```diff
@@ -19,7 +19,7 @@
 """)
 
 import ah_bootstrap
-from setuptools import setup
+from setuptools import setup, find_packages
```

Comment:
where does find_packages get called such that it needs to be imported here? Is it necessary for the recursive-include part to work? I have to admit I don't have a good understanding of the internals of setuptools

Reply:
Thats right we don't need it. I will remove that now. I found a lot of useful information/instructions on this [page](http://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files).